docs/circuit_cutting/tutorials/01_gate_cutting_to_reduce_circuit_width.ipynb

```diff
@@ -283,21 +283,26 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# from qiskit_ibm_runtime import Session, Options, Sampler\n",
+    "# from qiskit_ibm_runtime import Session, Options, Sampler, QiskitRuntimeService\n",
+    "# from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager\n",
     "#\n",
-    "# with Session(backend=\"ibmq_qasm_simulator\") as session:\n",
+    "# service = QiskitRuntimeService()\n",
+    "# backend = service.least_busy(operational=True, simulator=False)\n",
```

Comment:
Maybe we can remove the Runtime example in the comments here (and our other tutorials) in a future PR and use a fake backend from qiskit_ibm_runtime. The usage should be the same for the fake backend and real backend.

Reply:
Good call.  I opened #511 so we don't lose track of this.